src/datasource-zabbix/datasource.js

```diff
@@ -640,6 +640,15 @@ class ZabbixAPIDatasource {
     );
     return useTrends;
   }
+
+  setDescription(items, options) {
+    let panelModel = this.dashboardSrv.dash.getPanelById(options.panelId);
+    let desc = items.map(i => i.description);
+    desc = desc.length === 1 ? desc[0] : desc;
+    panelModel.scopedVars.description = {
```

Comment:
This cause error when `panelModel.scopedVars` is undefined. You should set it to default value before: 
```js
panelModel.scopedVars = panelModel.scopedVars || {}
```

Reply:
ok